package.json

```diff
@@ -41,7 +42,7 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "eth-sig-util": "^3.0.1",
-    "ethers": "^5.5.4",
+    "ethers": "5.5.4",
```

Comment:
Is this change by intention?

Reply:
Reverting this, seems I had an error but I can't reproduce it anymore, maybe it was my node version.